core/clab.go

```diff
@@ -105,50 +104,6 @@ func NewContainerLab(opts ...ClabOption) (*CLab, error) {
 	return c, err
 }
 
-// NewClabFromTopologyFileOrLabName creates a containerlab instance using either a topology file
-// path or a lab name. It returns the initialized CLab structure with the topology loaded.
-func NewClabFromTopologyFileOrLabName(
```

Comment:
removing this is prolly the highlight -- was confusing me why it was needed for a while :)